ci/deploy/image-repositories.yaml

```diff
@@ -0,0 +1,90 @@
+---
```

Comment:
If you're only deploying (and not building) you don't need any of these (see ci/prod or ci/integration)